packages/client/components/DashTopBar.tsx

```diff
@@ -35,9 +35,15 @@ const Wrapper = styled('header')({
 })
 
 const LeftNavToggle = styled(PlainButton)({
+  borderRadius: 100,
   fontSize: ICON_SIZE.MD24,
   lineHeight: '16px',
-  paddingLeft: 16
+  paddingLeft: 16,
+  margin: 8,
+  padding: 4,
+  ':focus': {
+    boxShadow: '0 0 0 2px #61B1EB'
```

Comment:
Instead of the hex for the color, use a template literal with the `SKY_400` with the constant from paletteV3 — this needs to be replaced in 3 other places below

Example
```
import {PALETTE} from '../styles/paletteV3'
...
boxShadow: `0 0 0 2px ${PALETTE.SKY_400}`
```